### DISCUSSION TITLE
Is there any discord servers for oc2?

### DISCUSSION THREAD

**Discussion by saryginrodion** [fnuecke/oc2#252] [Q&A] [answered]
I searched for forums or ds servers, but didn't found anything related to oc2 instead of this github.

**Comment by atirutw** [comment, marked as answer]
We use the original OC server for anything OC and even generally techy stuff. I don't have the invite link anymore and inviting is disabled, so you'll have to wait ig.

**Reply by walksanatora** [reply]
https://discord.gg/bYqKv7h
I am in the oc discord, here it is
